uk.co.compucorp.civicrm.hrcontactactionsmenu/tests/phpunit/CRM/HRContactActionsMenu/Hook/AddContactMenuActionsTest.php

```diff
@@ -2,15 +2,33 @@
 
 use CRM_HRContactActionsMenu_Hook_AddContactMenuActions as AddContactMenuActionsHook;
 use CRM_HRContactActionsMenu_Component_Menu as ActionsMenu;
+use CRM_HRContactActionsMenu_Component_Group as ActionsGroup;
+use Civi\Test\HookInterface as HookInterface;
 
 /**
  * Class CRM_HRContactActionsMenu_Hook_AddContactMenuActionsTest
  *
  * @group headless
  */
-class CRM_HRContactActionsMenu_Hook_AddContactMenuActionsTest extends BaseHeadlessTest {
+class CRM_HRContactActionsMenu_Hook_AddContactMenuActionsTest extends BaseHeadlessTest implements HookInterface {
 
-  public function testInvokeReturnsAnActionsMenuInstance() {
-    $this->assertInstanceOf(ActionsMenu::class, AddContactMenuActionsHook::invoke());
+  private $groupTitle;
+
+  public function setUp() {
+    CRM_Utils_Hook::singleton()->setHook('addContactMenuActions', array($this, 'hook_civicrm_addContactMenuActions'));
```

Comment:
Is this really necessary? I think that the HookInterface should be enough for civi to do this automatically

Reply:
I copied this from core tests, Will remove and see if it works.